mitxpro/settings.py

```diff
@@ -1023,14 +1025,14 @@
 PASSWORD_RESET_CONFIRM_URL = "password_reset/confirm/{uid}/{token}/"
 
 # Djoser library settings (see: http://djoser.readthedocs.io/en/stable/settings.html)
-DJOSER = {
-    "PASSWORD_RESET_CONFIRM_URL": PASSWORD_RESET_CONFIRM_URL,
-    "SET_PASSWORD_RETYPE": False,
-    "LOGOUT_ON_PASSWORD_CHANGE": False,
-    "PASSWORD_RESET_CONFIRM_RETYPE": True,
-    "PASSWORD_RESET_SHOW_EMAIL_NOT_FOUND": True,
-    "EMAIL": {"password_reset": "authentication.views.CustomPasswordResetEmail"},
-}
+# DJOSER = {
+#     "PASSWORD_RESET_CONFIRM_URL": PASSWORD_RESET_CONFIRM_URL,
+#     "SET_PASSWORD_RETYPE": False,
+#     "LOGOUT_ON_PASSWORD_CHANGE": False,
+#     "PASSWORD_RESET_CONFIRM_RETYPE": True,
+#     "PASSWORD_RESET_SHOW_EMAIL_NOT_FOUND": True,
+#     "EMAIL": {"password_reset": "authentication.views.CustomPasswordResetEmail"},
+# }
```

Comment:
Could you just delete this? it'll be in git history anyway.

Reply:
done